docs/examples/generatedCanonicalFields.json

```diff
@@ -0,0 +1,822 @@
+[
+  {
+    "label": "Org Name",
```

Comment:
We will probably want to figure out a naming convention / style guide for canonical fields.  I'd recommend we have a rule that we don't use shortened versions of words in our canonical field names (e.g. this should be "Organization Name" rather than "Org Name")

We might also say that labels should expand acronyms AND define them.  E.g. we might say that "SSN" should be "Social Security Number (SSN)".

Reply:
See https://github.com/PhilanthropyDataCommons/service/pull/103#issuecomment-1289402724